fullstack-test.sh

```
@@ -7,6 +7,19 @@ GREEN='\033[0;32m'
 YELLOW='\033[1;33m'
 NC='\033[0m' # No Color
 
+MB_VALIDATOR_STARTED_BY_US=false
+EPHEMERAL_VALIDATOR_STARTED_BY_US=false
+
+# Check if a port is in use
+check_port() {
+  local port=$1
+  if lsof -Pi :$port -sTCP:LISTEN -t >/dev/null 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)